integration_tests/twodut_oneotg_tests/bgp_triggered_gue/bgp_triggered_gue_test.go

```diff
@@ -150,38 +155,74 @@ var (
 	}
 )
 
-// configureATE configures port1 and port2 on the ATE.
-func configureATE(t *testing.T, ate *ondatra.ATEDevice) gosnappi.Config {
+// configureOTG configures port1 and port2 on the ATE.
```

Comment:
update comment to include what is being configured. also look like more than 2 ports are configured

Reply:
I left a general comment here and let the comments within the function talk about more details.